http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4126,6 +4126,13 @@ components:
         labels:
           type: object
           description: The labels of each app in the response whose key is in includeLabels in the request. Of type Map[String,String]
+        autodeleteThreshold:
```

Comment:
Nit, but could you put `autoDeleteEnabled` before `autoDeleteThreshold` to match the code (and also the logic)?

Reply:
It will brake if the api doc and the code are inconsistent, which they were. The code was listing `autoDeleteEnabled` first and then `autoDeleteThreshold` which is why I was asking to swap them in the api doc to match the code. However I see that in the Create App request from your previous PR, it is `autoDeleteThreshold` first followed by `autoDeleteEnabled`. So really I will leave which one goes first up to you, as long as it is consistent throughout the code and the api doc

Reply:
I see. I also would like to swap order.  If I also swap order in `CreateAppRequest`, would that break Leo?

Reply:
It might for people using the old client. I would not worry too much about the createapp request, it is outside of the scope of this PR